gunicorn.service

```diff
@@ -0,0 +1,15 @@
+[Unit]
+Description=gunicorn daemon
+After=network.target
+
+[Service]
+Type=notify
+# the specific user that our service will run as
+User=vagrant
+WorkingDirectory=/vagrant
+ExecStart=VENV_PATH/bin/gunicorn --error-logfile /vagrant/gunicorn_error.log --access-logfile /vagrant/gunicorn_access.log --bind 0.0.0.0:5000 "todo_app.app:create_app()"
```

Comment:
Minor, but here you're using `todo_app.app:create_app()` while in your VM you're assuming the path is through `wsgi.py` - it would make sense to align those (although I don't have a strong preference for which!)